docs/snippets/web-components/simple-page-implementation.ts.mdx

```diff
@@ -0,0 +1,39 @@
+```ts
+import { LitElement, html } from 'lit-element';
+
+@customElement('demo-document-screen')
+class DocumentScreen extends LitElement {
+  static get properties() {
+    return {
+      data: { type: Object },
+    }
+  }
+
+  @property({ type: Object })
+  data: {
+    user: Record<string, unknown>;
+    document: Record<string, unknown>;
+    subdocuments: Array<Record<string, unknown>>;
+  } = {}
```

Comment:
I can remember issues with assigning default values in web components, but that may not be true with decorators?

Reply:
Docs here: https://lit.dev/docs/api/decorators/#property